Example/JustLog/AppDelegate.swift

```diff
@@ -27,7 +36,42 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func applicationWillTerminate(_ application: UIApplication) {
         forceSendLogs(application)
     }
-
+    
+    func redactValues(message: String, loggerExeptionList: [ExceptionListResponse], matches: [NSTextCheckingResult]) -> String {
+        
+        var redactedLogMessage = message
+        
+        for match in matches.reversed() {
+            let key = match.range(at: 1)
+            let value = match.range(at: 2)
+            
+            let keyRange = Range(key, in: redactedLogMessage)!
+            let valueRange = Range(value, in: redactedLogMessage)!
+            
+            for exception in loggerExeptionList {
+                if exception.value == redactedLogMessage[valueRange] {
+                    return redactedLogMessage
```

Comment:
I think that I'm missing something, but I prefer to ask because it can be a potential issue.
We exit the function if we find that a match is included in the list of exceptions, but what happens to the matches that are after it? They are not going to be redacted, are they?
Shouldn't we just skip the current match and continue to run the forin matches?

Reply:
Ok... I noticed only now that this is in the example app 🤦